financial-connections/src/main/java/com/stripe/android/financialconnections/features/consent/ui/ConsentLogoHeader.kt

```diff
@@ -139,30 +129,37 @@ private fun debugPreviewBitmaps(
 private fun loadBitmaps(
     logos: List<String>,
     bitmapLoadSize: Int,
-    isVisible: MutableState<Boolean>
-): SnapshotStateList<ImageBitmap?> {
+): List<ImageBitmap?> {
     val stripeImageLoader = LocalImageLoader.current
-    val loadedImages = remember { mutableStateListOf<ImageBitmap?>() }
-    // Load images
-    logos.forEachIndexed { index, logo ->
-        LaunchedEffect(logo) {
-            stripeImageLoader.load(
-                logo,
-                bitmapLoadSize,
-                bitmapLoadSize
-            ).onSuccess { result ->
-                while (loadedImages.size <= index) {
-                    loadedImages.add(null)
+    val loadedImages = remember { mutableStateListOf(*Array<ImageBitmap?>(logos.size) { null }) }
+    val placeholderBitmap = generatePlaceholderBitmap(bitmapLoadSize, colors.textCritical)
+
+    LaunchedEffect(logos) {
+        logos.forEachIndexed { index, logo ->
+            stripeImageLoader.load(logo, bitmapLoadSize, bitmapLoadSize).let { loadResult ->
```

Comment:
Nit: We can call `onSuccess` and `onFailure` directly without the need for `let`.